quiz.js

```diff
@@ -62,8 +87,6 @@ db.dropDatabase();
 
 // your code begins here
 
-
-
-
+db.houses.remove({ name: 'Stark'}, { motto: 'Winter is Coming'});
```

Comment:
Just `db.houses.remove({ name: 'Stark' })` - that last bit with the motto is unnecessary and would be ignored.
-0.5